src/main/java/org/graylog/integrations/ipfix/codecs/IpfixCodec.java

```diff
@@ -108,6 +111,8 @@ protected IpfixCodec(@Assisted Configuration configuration, IpfixAggregator ipfi
         this.parser = new IpfixParser(this.infoElementDefs);
     }
 
+
```

Comment:
Some extra blank lines were also added in commit 584c4f1aadac4aa21aa74de6e9c1e77c0d4ba6b6. Can this be removed?